app/frontend/src/views/admin/Submission.vue

```diff
@@ -93,6 +99,10 @@ export default {
     ...mapActions('form', ['getForm']),
     refreshNotes() {
       this.$refs.notesPanel.getNotes();
+    },
+    operationTypeDisplayName(operationType){
+      const obj =  this.operationTypes.find( ({ type })  => type === operationType);
+      return obj.display;
```

Comment:
This is where the error from my comment in the main conversation is coming from. Just a ternary here can handle the edge case